### ISSUE TITLE
Leviathan should disable towers with his tentacles upon appearance.

### ISSUE THREAD

**Issue by sunnydaler** [cigumo/krli#741]
in iOS, he temporarily disables the towers along the shorelines.
![Screenshot_20190327-215316_YouTube (1)](https://user-images.githubusercontent.com/43391939/55078645-f6caef80-50dd-11e9-83ff-81723fd263da.jpg)

**Comment by cigumo** [comment]
Which wave? 15?

**Comment by sunnydaler** [comment]
Steam, Android: https://i.imgur.com/ThRpBJM.mp4
iOS: https://i.imgur.com/eaQ1qk2.mp4

**Comment by sunnydaler** [comment]
ah! Listen to how the music syncs with this sequence on iOS! 
boss emerges and stays there for a while, giving enough time for the boss emerging sfx to end. (in love, the boss starts moving before the sfx ends.)
And the music erupts with a bang when the boss starts to move and the battle resumes! How suspenseful!

Love needs a fix.

**Comment by Mmlh1** [comment]
I agree the boss music sync up makes it far more epic, and it would be great if the timing could be fixed.
